src/server/network/socket-gameplay.js

```diff
@@ -0,0 +1,55 @@
+const socketsHelper = require("./sockets-helper");
+const log = require("../../log");
+const db = require("../database/manager");
+
+const socketGameplay = function(getInitialDataPayload) {
```

Comment:
This currently isn't a module, but a constructor of some kind (in`game.js` it's called as a function to create an instance).
It's currently part of a circular dependency (hotfixed by passing a function as parameter), and could be resolved by changing the architecture: Network-code is to handle tickrate & when to emit messages (requesting the data from `game`), removing the need for the `game` module to refer to and/or manage any kind of networking.

Edit: It may actually stay instantiate-able (probably not a word but oh well) but leaving the full message here just in case.